docs/refs.bib

```diff
@@ -1468,4 +1468,10 @@ @article{Zhang_2008_Beyond_realignment
   url = {https://link.aps.org/doi/10.1103/PhysRevA.77.060301}
 }
 
+@misc{qetlab_commutant,
```

Comment:
The convention is to cite the entire `QETLAB` project. That entry already exists https://github.com/vprusso/toqito/blob/b9ed602cd3f224aa1c3335d13022d30c7e07b2a9/paper/paper.bib#L12